File: External/meegpipe/+mperl/+file/+spec/case_tolerant.m
function res = case_tolerant()
% CASE_TOLERANT - Indicates whether file names are case tolerant
%
%
% res = filespec.case_tolerant;
%
% Where
%
% RES is true (resp. false) if alphabetic case is no (resp. is) 
% significant when comparing file specifications.
%
%
% See also: filespec

% (c) German Gomez-Herrero, german.gomezherrero@kasku.org

% Documentation: pkg_filespec.txt
% Description: Indicates whether file names are case tolerant


res = eval(perl('+mperl/+file/+spec/case_tolerant.pl'));


end

File: External/meegpipe/+mperl/+file/+spec/case_tolerant.pl
#!/usr/bin/perl
# (c) German Gomez-Herrero, german.gomezherrero@kasku.org

# Description:  Indicates whether file names are or not case sensitive
# Documentation: pkg_filespec.txt

use warnings;

# Make our Spec.pm visible
my $libLocation;
BEGIN {
    use Cwd 'abs_path';
    use File::Spec;
    my $name = abs_path($0);
    $name=~m/^.+\//; 
    $libLocation = abs_path(File::Spec->catdir($&, '../../lib'));
}

use lib $libLocation;

use File::Spec;

my $res = File::Spec->case_tolerant();

if ($res){
    print 'true';
} else { 
    print 'false';
}

